api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/annotations/annotations.md

```diff
@@ -8,7 +8,7 @@ inherits: dxChart.Options.commonAnnotationSettings
 Specifies the annotation collection.
 
 ---
-Annotations are images and text blocks that provide additional information on the visualized data. The image below demonstrates their appearance:
+Annotations are containers for images, text blocks, and custom content that provide additional information about the visualized data. The image below demonstrates their appearance:
```

Comment:
"The image below demonstrates their appearance:" I do not think it is necessary to add this information.